src/docs/release/breaking-changes/android-v1-embedding-create-deprecation.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Android v1 embedding app and plugin creation deprecation
+description: Gradual deprecation of the Android v1 embedding
+---
+
+## Summary
+
+The `flutter create` templates for apps and plugins will no longer create
+Android wrapping based on the v1 Android embedding as part of our gradual
+Android v1 embedding deprecation process described in
+https://flutter.dev/go/android-migration-summary.
+
+Application projects using the v1 Android embedding are encouraged to migrate
+following https://flutter.dev/go/android-project-migration.
+
+Plugins targeting the v1 Android embedding are encouraged to migrate following
+https://flutter.dev/go/android-plugin-migration.
+
+## Context
+
+In Flutter version 1.12, we launched a v2 set of Android APIs based in
+https://cs.opensource.google/flutter/engine/+/master:shell/platform/android/io/flutter/embedding/
+in order to enable the [add-to-app](https://flutter.dev/docs/development/add-to-app)
+workflow on Android.
+
+Over time, we're gradually deprecated the older v1 Android embeddings based on
+https://cs.opensource.google/flutter/engine/+/master:shell/platform/android/io/flutter/app/.
+
+In Q2 2020, only 26% of applications are using the v1 embeddings.
+
+Since the v2 embeddings have been strongly established over the 7 months since
```

Comment:
have been => were

(Remember to write this breaking change as if it already happened.)